gradlew

```diff
@@ -55,7 +55,7 @@
 #       Darwin, MinGW, and NonStop.
```

Comment:
Huh, so the Gradle Wrapper does have updates that aren't tied to any Gradle version; interesting

Reply:
Correct - This must be updated manually aswell since gradle cannot do automatically as such automation doesn't exist yet iirc.